draft-ietf-quic-transport.md

```diff
@@ -3408,9 +3408,14 @@ Google QUIC Demultiplexing Bit:
   specification when Google QUIC has finished transitioning to the new header
   format.
 
+Spin Bit:
+
+: The sixth bit (0x4) of byte 0 is the Latency Spin Bit, set as described in
+{{!SPIN=I-D.ietf-quic-spin-exp}}.
+
 Reserved:
 
-: The sixth, seventh, and eighth bits (0x7) of byte 0 are reserved for
+: The seventh and eighth bits (0x3) of byte 0 are reserved for
```

Comment:
I don't think we're reserving any more bits for experimentation at this point.

Reply:
By un-reserving them you're shorting any hope for other measurement, like loss bits. DO you decide this kind of things single-handedly ?